class UserAlreadyExists(Exception):
    pass

class UserNotFound(Exception):
    pass